package.json

```
@@ -15,8 +15,8 @@
   "author": "Gregor Martynus (https://twitter.com/gr2m)",
   "license": "ISC",
   "scripts": {
-    "lint": "prettier --check 'lib/*.js' cli.js README.md package.json",
-    "lint:fix": "prettier --write 'lib/*.js' cli.js README.md package.json",
+    "lint": "prettier --check 'lib/*.js' cli.js **/*.md **/*.json",
+    "lint:fix": "prettier --write 'lib/*.js' cli.js **/*.md **/*.json",
```

you have to wrap the glob patterns in quotes, otherwise they can be resolved by the shell you are using before being passed to prettier
```suggestion
    "lint": "prettier --check 'lib/*.js' cli.js '**/*.md' '**/*.json'",
    "lint:fix": "prettier --write 'lib/*.js' cli.js '**/*.md' '**/*.json'",
```